wgsl/index.bs

```diff
@@ -13383,21 +13384,21 @@ Note: The vec2&lt;f32&gt; case is the same as `unpack2x16float(pack2x16float(e))
 
 ### `sign` ### {#sign-builtin}
 <table class='data builtin'>
-  <tr algorithm="float sign">
+  <tr algorithm="numeric sign">
     <td style="width:10%">Overload
     <td class="nowrap"><xmp highlight=rust>
 @const fn sign(e: T) -> T
 </xmp>
   <tr>
     <td style="width:10%">Parameterization
-    <td>[ALLFLOATINGDECL]
+    <td>[ALLSIGNEDNUMERICDECL]
```

Comment:
It looks like you could use this for Negation in [Unary arithmetic expressions](https://www.w3.org/TR/WGSL/#arithmetic-expr)